build.gradle

```diff
@@ -2,17 +2,19 @@
 
 buildscript {
     repositories {
+        google()
```

Comment:
This line produces the following error in Studio 2.3.3:
```Gradle DSL method not found: 'google()'```
Are you running the 3.0 beta, or do I have to check my Gradle config?